flake.lock

```diff
@@ -321,6 +354,32 @@
         "type": "github"
       }
     },
+    "devenv_2": {
+      "inputs": {
+        "flake-compat": [
+          "common",
```

Comment:
Also this?

So many unnecessary changes.

You should revert the whole `flake.lock` change, and then update it correctly using `nix flake lock`.

Reply:
sure, this PR is for syncing the last prod release though, not for the redis change.